### PR TITLE
Production Readiness

### LINKED ISSUE #41: TransitMatters logo in chart doesn't show up well in Dark Mode
Good
image

Bad
image

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

Ready for production

Fixes #41 #44 #38 #43 #40

## Changes

- Updates unfurl metadata for sites
- Adds manifest for mobile
- Updates dark mode logo
- Fixes routing
- Fixes content type on calendar
- Adds footer with donate link
- Updates codeowners for prod deploy
- Adds goatcounter

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "short_name": "Shutdown Tracker",
+  "name": "Transitmatters MBTA Shutdown Tracker",
```
Comment (nathan-weinberg) [inline public/manifest.json:3]:
```suggestion
  "name": "TransitMatters MBTA Shutdown Tracker",
```

File: src/constants/site.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const PRODUCTION = 'mbtashutdowns.info';
```
Comment (nathan-weinberg) [inline src/constants/site.ts:1]: I think @lawrencet1 wanted us to also have a TransitMatters subdomain as well as this one? @idreyn can we replicate what we did for the Regional Rail Explorer where we have both?

**Review by nathan-weinberg — COMMENTED** [review]
Great stuff @devinmatte! Couple small comments

File: src/constants/site.ts (hunk 2 above)
Comment (devinmatte) [reply src/constants/site.ts:1]: Is there another one for the regional rail explorer? If so I can't find a config, dns record or link to it

Comment (nathan-weinberg) [reply src/constants/site.ts:1]: There definitely was at one point, though I'm struggling to remember what it was - maybe we actually replaced it

Comment (idreyn) [reply src/constants/site.ts:1]: I don't recall us ever having an RRE subdomain

Comment (nathan-weinberg) [reply src/constants/site.ts:1]: Alright I say we just proceed with this then

**Review by nathan-weinberg — APPROVED** [review]

File: src/constants/site.ts (hunk 2 above)
Comment (lawrencet1) [reply src/constants/site.ts:1]: We had talked about shutdowns.transitmatters.org on the call.

Comment (devinmatte) [reply src/constants/site.ts:1]: Yep we'll still add it, just need to figure out the right way to do it. Looking into it tonight

**Pull request merged**